docker/test.Dockerfile

```diff
@@ -21,4 +21,7 @@ COPY predict_payload.json .
 COPY predict_result.json .
 COPY components/* components/
 
+ARG MODEL_NAME_APPEND=""
```

Comment:
I think we need a rebase. The latest version doesn't require --config in test.Dockerfile. We can just create a new json file.

Reply:
Discussed offline - will handle this in follow-up PR.